rotkehlchen/tests/unit/decoders/test_zerox.py

```diff
@@ -1215,7 +1215,7 @@ def test_swap_anon_event(ethereum_inquirer: 'EthereumInquirer', ethereum_account
         location=Location.ETHEREUM,
         event_subtype=HistoryEventSubType.SPEND,
         asset=Asset('eip155:1/erc20:0x58D97B57BB95320F9a05dC918Aef65434969c2B2'),
-        amount=FVal(swap_amount := '7734.443526456381826329'),
+        amount=FVal(swap_amount := '7720.393471610974409838'),
```

Comment:
why did this change now?

Reply:
It is a 0x swap using cowswap and after adjusting the logic to take into account the fee part of the spent amount went to the new fee attribute a few lines below.